index.dd

```diff
@@ -321,8 +321,8 @@ $(DIVC org_list,
 
 $(DIVC org_list_footer,
     <p class="center">
-        $(LINK2 $(ROOT_DIR)orgs-using-d.html, Testimonials And More Organizations)
-        $(LINK2 $(ROOT_DIR)areas-of-d-usage.html, Current D use)
+        $(FA_ICON quote-left) $(LINK2 $(ROOT_DIR)orgs-using-d.html, Testimonials and more organizations)
```

Comment:
I'm not sure on the last change anymore. Looking at both screenshots both options seem fine, so I'm happy to revert this case style change.